src/core/worker.js

```diff
@@ -555,12 +452,7 @@ var WorkerMessageHandler = {
             !fullRequest.isRangeSupported) {
           // If stream or range are disabled, it's our only way to report
           // loading progress.
-          fullRequest.onProgress = function (evt) {
-            handler.send('DocProgress', {
-              loaded: evt.loaded,
-              total: evt.total,
-            });
-          };
+          handler.send('DocProgress');
```

Comment:
Removed with new commit, as we don't need this anymore. As we are moving all networking related tasks to main side, we can easily notify `DocProgress` at main side in [ReaderHeaderReady](https://github.com/mozilla/pdf.js/pull/8617/commits/4531fea269ca195d6a72a4ab45ff00629b00fe58#diff-0ecad279ed2252e3eb47a4d96ec1463cR1625), and eliminates round trip of progress data.